ops/terraform/services/common/app-autoscaling.tf

```diff
@@ -20,37 +29,73 @@ resource "aws_appautoscaling_target" "replicas" {
   }
 }
 
-resource "aws_appautoscaling_scheduled_action" "scale_out" {
-  count = local.env == "test" ? 1 : 0
+resource "aws_appautoscaling_target" "static_replicas" {
+  depends_on = [aws_rds_cluster_instance.writer]
+
+  # only applicable for prod/prod-sbx (not test or ephemeral)
+  count = !local.enable_rds_scheduled_scaling ? 1 : 0
+
+  service_namespace  = "rds"
+  scalable_dimension = "rds:cluster:ReadReplicaCount"
+  resource_id        = "cluster:${aws_rds_cluster.aurora_cluster.id}"
+
+  min_capacity = local.rds_min_reader_nodes
+  max_capacity = local.rds_max_reader_nodes
+}
 
-  name               = "bfd-${local.env}-scale-out"
-  service_namespace  = aws_appautoscaling_target.replicas[0].service_namespace
-  resource_id        = aws_appautoscaling_target.replicas[0].resource_id
-  scalable_dimension = aws_appautoscaling_target.replicas[0].scalable_dimension
+resource "aws_appautoscaling_policy" "replicas_cpu_scaling" {
+  depends_on = [aws_rds_cluster_instance.writer]
+
+  name               = "bfd-${local.env}-cpu-scaling"
+  service_namespace  = local.replicas_scaling_target.service_namespace
+  resource_id        = local.replicas_scaling_target.resource_id
+  scalable_dimension = local.replicas_scaling_target.scalable_dimension
+  policy_type        = "TargetTrackingScaling"
+
+  target_tracking_scaling_policy_configuration {
+    predefined_metric_specification {
+      predefined_metric_type = "RDSReaderAverageCPUUtilization"
+    }
+
+    target_value       = local.rds_scaling_cpu_target
+    scale_in_cooldown  = local.rds_scale_in_cooldown
+    scale_out_cooldown = local.rds_scale_out_cooldown
+  }
+}
+
+resource "aws_appautoscaling_scheduled_action" "work_hours_scale_out" {
+  depends_on = [aws_rds_cluster_instance.writer]
+
+  count = local.enable_rds_scheduled_scaling ? 1 : 0
+
+  name               = "bfd-${local.env}-work-hours-scale-out"
+  service_namespace  = local.replicas_scaling_target.service_namespace
+  resource_id        = local.replicas_scaling_target.resource_id
+  scalable_dimension = local.replicas_scaling_target.scalable_dimension
   schedule           = "cron(00 07 ? * MON-FRI *)"
```

Comment:
@aschey-forpeople brought up a good point during demo: for contributors working outside of ET there may be cases where this `cron` schedule (and the corresponding scale-in `cron`) _do_ overlap with 9-5 working hours. Should we consider modifying these schedules to make them a _bit_ more flexible? Perhaps something like 6 AM ET for scale-out (this scheduled action) and 8 PM ET for scale-in?

Reply:
On the other hand, I think in the (probably _very_) exceptional case where testing needs to happen against an ephemeral cluster _with_ autoscaling enabled _outside_ of the existing 7 AM - 7 PM ET window could be served by simply enabling the `disable_rds_scheduling_override`.

Reply:
Yeah, that's fine. Considering this won't be something we need to mess with frequently, redeploying `common` with that parameter seems perfectly acceptable.